ci-operator/config/cri-o/cri-o/cri-o-cri-o-release-1.13.yaml

```diff
@@ -0,0 +1,69 @@
+base_images:
+  base:
+    name: "4.1"
+    namespace: ocp
+    tag: base
+  base-machine:
+    cluster: https://api.ci.openshift.org
+    name: fedora
+    namespace: openshift
+    tag: "29"
+  machine-os-content-base:
+    name: "4.1"
+    namespace: ocp
+    tag: machine-os-content
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.11
+canonical_go_repository: github.com/cri-o/cri-o
+images:
+- context_dir: images/os/
+  from: base
+  inputs:
+    base-machine-with-rpms:
+      as:
+      - fedora:29
+      paths: null
+    machine-os-content-base:
+      as:
+      - registry.svc.ci.openshift.org/openshift/origin-v4.1:machine-os-content
+      paths: null
+  to: machine-os-content
+promotion:
+  excluded_images:
+  - machine-os-content
+  name: "4.1"
+  namespace: ocp
+raw_steps:
+- rpm_image_injection_step:
+    from: base-machine
+    to: base-machine-with-rpms
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: 100m
+      memory: 4Gi
+rpm_build_commands: hack/build-rpms.sh
+tag_specification:
+  name: "4.1"
+  namespace: ocp
+tests:
+- as: unit
+  commands: make testunit
+  container:
+    from: src
+- as: e2e-aws
+  commands: TEST_SUITE=openshift/conformance/parallel run-tests
+  openshift_installer:
+    cluster_profile: aws
+    upgrade: false
+- as: launch-aws
```

Comment:
you'll need to make this an optional job (after generating) by adding the correct "optional" attributes in presubmits.yaml.  See what origin launch-aws does. (required: false, optional: true, I think)

Reply:
done